<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WarHack v1.2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resources.qrc">
    <normaloff>:/images/iconfinder__snowflake_1679761_ico.ico</normaloff>:/images/iconfinder__snowflake_1679761_ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonStart">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  border-radius: 28px;
  font-family: Arial;
  color: #ffffff;
  font-size: 20px;
  background: #00c4ff;
  padding: 5px 10px 5px 10px;
  text-decoration: none;
}

QPushButton:hover {
  background: #04b52d;
  text-decoration: none;
}</string>
      </property>
      <property name="text">
       <string>Iniciar</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="buttonSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  border-radius: 28px;
  font-family: Arial;
  color: #ffffff;
  font-size: 20px;
  background: #00c4ff;
  padding: 5px 10px 5px 10px;
  text-decoration: none;
}

QPushButton:hover {
  background: #04b52d;
  text-decoration: none;
}</string>
      </property>
      <property name="text">
       <string>Configurações</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
	font: 20px &quot;Arial&quot;;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 20px;
    border-top-right-radius: 20px;
    padding: 5px 150px;
    background-color: #00C4FF;
    color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="title">
          <string>Console</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="textBrowserOutput">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionQuit_2"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_WarHack"/>
    <addaction name="actionCheckUpdates"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="icon">
    <iconset resource="../resource/resources.qrc">
     <normaloff>:/images/if_Log Out_27856.ico</normaloff>:/images/if_Log Out_27856.ico</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../resource/resources.qrc">
     <normaloff>:/images/if_qt_337140.ico</normaloff>:/images/if_qt_337140.ico</iconset>
   </property>
   <property name="text">
    <string>Sobre o Qt</string>
   </property>
  </action>
  <action name="actionAbout_WarHack">
   <property name="icon">
    <iconset resource="../resource/resources.qrc">
     <normaloff>:/images/if_Blue_ball_132060.ico</normaloff>:/images/if_Blue_ball_132060.ico</iconset>
   </property>
   <property name="text">
    <string>Sobre o Spark</string>
   </property>
  </action>
  <action name="actionPortuguese">
   <property name="text">
    <string>Português</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionCheckUpdates">
   <property name="icon">
    <iconset>
     <normaloff>../resource/images/icon_update.ico</normaloff>../resource/images/icon_update.ico</iconset>
   </property>
   <property name="text">
    <string>Buscar atualizações</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/resources.qrc"/>
  <include location="../resource/resources.qrc"/>
 </resources>
 <connections/>
</ui>
